test/projections.erl

```diff
@@ -671,3 +671,76 @@ unregister_projection_test_() ->
             {error, ?khepri_error(projection_not_found, _Info)},
             khepri:unregister_projection(?FUNCTION_NAME, ?MODULE))}]
       }]}.
+
+unregister_all_projections_test_() ->
+    ProjectFun = fun(Path, Payload) -> {Path, Payload} end,
+    PathPattern = [stock, wood, <<"oak">>],
+    Data = 100,
+    ProjectionName1 = projection_1,
+    ProjectionName2 = projection_2,
+    {setup,
+     fun() -> test_ra_server_helpers:setup(?FUNCTION_NAME) end,
+     fun(Priv) -> test_ra_server_helpers:cleanup(Priv) end,
+     [{inorder,
```

Comment:
Could you please test that `unregister_all_projections()` works with no projections at all?